README.md

```diff
@@ -1,9 +1,2 @@
-# microservice-template
```

Comment:
Add description of the endpoints in readme. I'm not sure what each endpoint does just from reading the urls